modules/mongodb/testdata/etc/mongod.conf.obfuscated

```diff
@@ -0,0 +1,38 @@
+#System Log
+
+systemLog.path: /var/log/mongodb/mongod.log
+systemLog.destination: file
+systemLog.logAppend: true
+systemLog.quiet: false
+
+#Process Management
+processManagement:
+  pidFilePath: /var/run/mongodb/mongod.pid
+
+#Storage
+storage.dbPath: /var/lib/mongodb
+storage.journal.enabled: true
+
+#Security
+security.authorization: disabled
+security.example.foo<HIDDEN>: <HIDDEN>
+
+#Net
+net:
+   port: <int>
+   bindIp: <string>
+   bindIpAll: <boolean>
+   maxIncomingConnections: <int>
+   wireObjectCheck: <boolean>
+   ipv6: <boolean>
+   unixDomainSocket:
+      enabled: <boolean>
+   tls:
+      certificateKeyFile<HIDDEN>: <HIDDEN>
```

Comment:
Looks like the regex filter still has a problem here. The key itself should not be hashed.
Could you take another look at it here?

Reply:
Done